package.json

```diff
@@ -58,27 +59,23 @@
     "node": ">=0.10"
   },
   "dependencies": {
-    "assert-plus": "^0.1.5",
-    "backoff": "^2.4.0",
+    "assert-plus": "0.1.5",
```

Comment:
My thinking was "they've released 0.2 so I don't see them releasing more versions or 0.1—therefore saying *exactly* 0.15 communicates the best most accurate information". No problem with throwing the ~ in there though.